qdrant/v0.10.x/search.md

```diff
@@ -194,6 +191,111 @@ client.search(
 )
 ```
 
+## Batch search API
+
+*Available since v0.10.0*
+
+The batch search API enables to perform multiple search requests via a single request.
+
+Its semantic is straight-forward, `n` batched search requests are equivalent to `n` singular search requests.
+
+This approach has several advantages. First fewer network connections are required which can be very beneficial.
+
+More importantly, batched requests will be efficiently processed via the query planner which can detect and optimize requests if they have the same `query_filter`.
```

Comment:
You started counting the advantages. Here is better to write `Second, more importantly`. Or delete `First` earlier